Applications/Games/Hearthstone/Online/script.json

```diff
@@ -2,11 +2,13 @@
     "scriptName"                 : "Online",
     "id"                         : "online",
     "compatibleOperatingSystems" : [
-        "LINUX"
+        "LINUX",
+        "MACOSX"
     ],
     "testingOperatingSystems" : [
-        "LINUX"
+        "LINUX",
+        "MACOSX"
     ],
     "free"          : true,
-    "requiresPatch" : false
+    "requiresPatch" : true
```

Comment:
It should stay false here.

Reply:
@Zemogiter elaborate?

Reply:
@Kreyren from https://phoenicisorg.github.io/scripts/Develop/script-json/

> Application cannot work without patching the original protection. It may be illegal to install it depending on your country.

That means this variable should be true only if the game requires patching the copy protection a.k.a replacing the .exe or .dll file with a different one. The best example would be my Lego Rock Raiders script.

Reply:
@Zemogiter noted

@ImperatorS79 Can you finish the commit/update the script since you made it?